src/sentry/workflow_engine/processors/delayed_workflow.py

```diff
@@ -389,6 +390,7 @@ def fire_actions_for_groups(
     trigger_type_to_dcg_model: dict[DataConditionHandler.Type, dict[int, int]],
     group_to_groupevent: dict[Group, GroupEvent],
 ) -> None:
+    organization: Organization | None = None
```

Comment:
where are you passing org into this function?

Reply:
oops